src/backend/src/services/users.services.ts

```diff
@@ -538,4 +539,26 @@ export default class UsersService {
 
     return userScheduleSettingsTransformer(newUserScheduleSettings);
   }
+
+  static async getUserTasks(userId: string, organization: Organization) {
+    const requestedUser = await prisma.user.findUnique({ where: { userId } });
+    if (!requestedUser) throw new NotFoundException('User', userId);
+
+    const tasks = prisma.task.findMany({
```

Comment:
We can find the user then get assigned tasks from there. Though this might be trivial in reality and if this works then we can keep it as is. What are your thoughts?